plaid-stl/src/npm/mod.rs

```diff
@@ -0,0 +1,607 @@
+use serde::{Deserialize, Deserializer, Serialize};
```

Comment:
This whole file is calling out for some macros to remove a lot of the boilerplate. Might be something to consider doing here then applying it to the rest of the STL.